lib/team.rb

```diff
@@ -18,5 +18,58 @@ def team_info(team_id)
       link: "link"
     }
     team_info.transform_keys! { |k| key_map[k] }
+
```

Comment:
I messed up, think we need an end statement here on line 21. We can both do this locally if you're cool with the merge.